### ISSUE TITLE
Trivial accessibility fixes

### ISSUE THREAD

**Issue by wrightmartin** [mysociety/alaveteli#3121]
These are accessibility fixes for alaveteli core, and alaveteli theme that can be done without discussion. Part of #3095 

- [x] Missing form label on feedback box
- [x] Search bar on authorities page is missing a label
- [x] Missing form label on select_authority search
- [x] Numbers in the ‘how it works’ section should come before the text snippet, or not at all
- [x] Contrast of the ‘status’ text on the request page is too low
- [x] Orange colour is too low contrast for text
- [x] White on green colour is too low contrast for text

**Comment by garethrees** [comment]
Both PRs merged
